Sources/Interaction/Manipulators/MouseRangeManipulator/index.js

```diff
@@ -168,6 +196,18 @@ function vtkMouseRangeManipulator(publicAPI, model) {
 
   publicAPI.onButtonUp = (interactor) => {
     interactor.exitPointerLock();
+
+    if (model.scrollListener) {
```

Comment:
could store this on the model instead to cut down on repeated code? Not sure what convention is here.